providers/meteoblue.go

```diff
@@ -0,0 +1,74 @@
+package providers
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/tinkershack/meteomunch/config"
+	"github.com/tinkershack/meteomunch/http/rest"
+	"github.com/tinkershack/meteomunch/plumber"
+)
+
+const meteoBlueProviderName = "meteoblue"
+
+type MeteoBlueProvider struct {
+	client rest.HTTPClient
+	config config.MeteoProvider
+}
+
+func NewMeteoBlueProvider() (*MeteoBlueProvider, error) {
+	cfg, err := config.New()
+	if err != nil {
+		return nil, err
+	}
+
+	var meteoConfig config.MeteoProvider
+	found := false
+
+	for _, provider := range cfg.MeteoProviders {
+		if provider.Name == meteoBlueProviderName {
+			meteoConfig = provider
+			found = true
+			break
+		}
+	}
+
+	if !found {
+		return nil, errors.New("meteoblue provider configuration not found")
+	}
+
+	return &MeteoBlueProvider{
+		client: rest.NewClient(),
+		config: meteoConfig,
+	}, nil
+}
+
+func (p *MeteoBlueProvider) FetchData(qp map[string]string) (*plumber.BaseData, error) {
+	resp, err := p.client.SetQueryParams(map[string]string{
+		"lat": "11.25",
+		"lon": "77",
+	}).Get("https://my.meteoblue.com/v1/forecast")
```

Comment:
I don't think client had a new request made here(unless I'm missing something), will this work?

Reply:
Sorry, i seem to have forgotten to drop a comment in this file stating that it's just a placeholder. However, I have called it out in the commit message.